content/docs/iac/clouds/kubernetes/_index.md

```diff
@@ -13,7 +13,7 @@ menu:
 cloud_overview: true
 description: Streamline Kubernetes cluster configuration, management, and application workload deployments using TypeScript, Python, Go, C#, Java or YAML. Use the Pulumi Kubernetes Operator to manage both Kubernetes and cloud resources.
 get_started_guide:
-  link: get-started/
```

Comment:
I would _love_ to tackle this issue someday: https://github.com/pulumi/docs/issues/11581